aten/src/THCUNN/THCHalfAutoNumerics.cuh

```diff
@@ -32,7 +32,7 @@ inline __host__ __device__ double fmaxType(double x, double y) {
 // arithmetic functions
 
 inline __host__ __device__ THHalf exp(THHalf a) {
```

Comment:
You sure you need to keep this function at all?

Reply:
Thanks. Missed removing it.